cargo-near/src/commands/deploy/mod.rs

```diff
@@ -0,0 +1,134 @@
+    {
+        let mut clap_variant = optional_clap_variant.unwrap_or_default();
+
+        let build_command_args =
+            if let Some(cli_build_command_args) = &clap_variant.build_command_args {
+                build_command::BuildCommand {
+                    release: cli_build_command_args.release,
+                    embed_abi: cli_build_command_args.embed_abi,
+                    doc: cli_build_command_args.doc,
+                    no_abi: cli_build_command_args.no_abi,
+                    out_dir: cli_build_command_args.out_dir.clone(),
+                    manifest_path: cli_build_command_args.manifest_path.clone(),
+                    color: cli_build_command_args.color.clone(),
+                }
+            } else {
+                build_command::BuildCommand::default()
+            };
+
+        if clap_variant.contract_account_id.is_none() {
+            clap_variant.contract_account_id = match Self::input_contract_account_id(&context) {
+                Ok(Some(contract_account_id)) => Some(contract_account_id),
+                Ok(None) => return interactive_clap::ResultFromCli::Cancel(Some(clap_variant)),
+                Err(err) => return interactive_clap::ResultFromCli::Err(Some(clap_variant), err),
+            };
+        }
+        let contract_account_id = clap_variant
+            .contract_account_id
+            .clone()
+            .expect("Unexpected error");
+
+        let new_context_scope = InteractiveClapContextScopeForContract {
+            build_command_args,
+            contract_account_id,
+        };
+
+        let output_context =
+            match ContractContext::from_previous_context(context, &new_context_scope) {
+                Ok(new_context) => new_context,
+                Err(err) => return interactive_clap::ResultFromCli::Err(Some(clap_variant), err),
+            };
+
+        match InitializeMode::from_cli(clap_variant.initialize.take(), output_context.into()) {
+            interactive_clap::ResultFromCli::Ok(initialize) => {
+                clap_variant.initialize = Some(initialize);
+                interactive_clap::ResultFromCli::Ok(clap_variant)
+            }
+            interactive_clap::ResultFromCli::Cancel(optional_initialize) => {
+                clap_variant.initialize = optional_initialize;
+                interactive_clap::ResultFromCli::Cancel(Some(clap_variant))
+            }
+            interactive_clap::ResultFromCli::Back => interactive_clap::ResultFromCli::Back,
+            interactive_clap::ResultFromCli::Err(optional_initialize, err) => {
+                clap_variant.initialize = optional_initialize;
+                interactive_clap::ResultFromCli::Err(Some(clap_variant), err)
+            }
+        }
+    }
+}
+
+impl Contract {
+    #[allow(dead_code)]
+    fn input_build_command_args(
+        _context: &near_cli_rs::GlobalContext,
+    ) -> color_eyre::eyre::Result<Option<build_command::BuildCommand>> {
+        Ok(None)
+    }
```

Comment:
Given that `from_cli` is implemented manually, and it does not call `input_build_command_args`, I believe you can remove this function.

Reply:
Done